ROADMAP.md

```diff
@@ -0,0 +1,28 @@
+# vision
+The initial vison for the CILG was to create place where anyone could learn about what is needed to host their own p2p infrastructure. It's less about protocols, apps, and products and more about identifying the best practices for managing/running hardware in the home, in the community or in the cloud.
+
+# strategy
+Identify one use case from the p2p world and provide a happy path to easily setup the infrastructure needed to support that use case. This will include scripts, documentation in many forms i.e. traditional docs, blog style guides, video walk-throughs, online meetups etc etc. (Initially this will focus on SSB Pub setup). Once the CILG has worked through this process with this narrow use case, start identifying other use cases and replicate the process.
```

Comment:
```suggestion
Identify one use case from the p2p world and provide a happy path to easily set up the infrastructure needed to support that use case. This will include scripts, documentation in many forms, i.e. traditional docs, blog style guides, video walk-throughs, online meetups, etc. (initially this will focus on setting up a SSB Pub). Once the CILG has worked through this process with this particular use case, we will start identifying other use cases and by replicating our learned process.
```